documents/retail-operations/brokering/scenarios.md

```diff
@@ -147,15 +147,15 @@ Retailers want to ensure that orders are only fulfilled from facilities within a
 
 <figure><img src="../.gitbook/assets/Proximity based routing.png" alt=""><figcaption><p>Proximity Based Routing</p></figcaption></figure>
 
-## Scenario 7: Split Shipping Threshold Logic
+## Scenario 7: Minimum Inventory Availability
 
-Retailers want to implement a split shipping threshold, allowing orders to be fulfilled from multiple locations if the inventory at a single location falls below a certain threshold. This approach helps maintain adequate inventory levels for walk-in customers while ensuring timely fulfillment.
+Retailers are looking to maintain sufficient in-store inventory for walk-in customers while still fulfilling online orders efficiently. To achieve this, they require a solution that prioritizes facilities with adequate stock during the order fulfillment process. By utilizing the Brokering Safety Stock feature, orders are routed only to locations with a minimum threshold of inventory, ensuring that critical stock levels are preserved for in-store customers.
```

Comment:
they require a solution that prioritizes facilities with adequate stock during the order fulfillment process.

Here we are not talking about prioritizing, we are saying that allocate only to those locations where the mi nimum stock is available or don't allow.

Also ask chatgpt to remove unnecessary adjectives

Reply:
also in sceneario 7 we also talked about adding line in partial allocation lines